src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts.mdx

```diff
@@ -171,31 +172,11 @@ Besides the standard controls you'd expect from a complete alerting solution, we
 
     <tr>
       <td>
-        Webhooks
+        Issue scoping and rollups
       </td>
 
       <td>
-        Customizable [webhooks](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/customize-your-webhook-payload) allow you to define custom headers, basic authentication, custom payloads, and more.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Incident scoping and rollups
-      </td>
-
-      <td>
-        Every alert policy can be configured to use one of [three violation grouping strategies](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/specify-when-new-relic-creates-incidents) to control the number of alert incidents created, and therefore the number of notifications sent.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Cross-product events
-      </td>
-
-      <td>
-        A dedicated [**Events** page](/docs/alerts/new-relic-alerts-beta/reviewing-events/review-events-across-products) that shows operational events across all of your products.
+        Every alert policy can be configured to use one of [three violation grouping strategies](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/specify-when-new-relic-creates-incidents) to control the number of alert issues created, and therefore the number of notifications sent.
```

Comment:
The linked page has a bad path.  It's not a beta feature and incidents have been replaced by issues.  It should be moved.

Reply:
Sounds good!